src/main.rs

```diff
@@ -383,6 +391,16 @@ fn collect_files(
     in_out_pairs
 }
 
+#[cfg(windows)]
+fn apply_glob_pattern(path: PathBuf) -> Vec<PathBuf> {
+    let input_path = path.to_str().unwrap();
+
+    glob::glob(input_path)
+        .expect("Failed to read glob pattern")
+        .map(|path| path.unwrap())
```

Comment:
If you change this to `flatten` it can skip errors without panicking.
```suggestion
        .flatten()
```

Reply:
Updated in e87920b